src/main/client/player/other-player-sprite.ts

```diff
@@ -68,8 +69,14 @@ export class OtherPlayerSprite extends Phaser.GameObjects.Container {
 
    private updatePosition(): void {
       this.positionInterpolation.step();
-      const position = this.positionInterpolation.get();
-      this.setPosition(position.x, position.y);
+      const centerPoint = this.positionInterpolation.get();
+      const bottomPosition = new Phaser.Math.Vector2({
+         x: centerPoint.x,
+         y: centerPoint.y,
+      }).add(
+         VectorUtil.getUpwardVector(this).scale(-ClientConfig.PLAYER_SPRITE_HEIGHT / ClientConfig.MAP_OUTPUT_SCALE),
```

Comment:
if the upward vector is scaled with a negative value, then it will point downwards

Are you sure this is correct? can you test? I have a feeling the upward vector should still point up

Reply:
No because from center to bottom position you need the opposite what we did in player-sprite.getCenterPoint() but please run and test it. The other player should show up in the correct position.

Reply:
oh i see, ok fine